seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -1769,9 +1769,10 @@
         "Each of your radio buttons should be nested within its own <code>label</code> element.",
         "All related radio buttons should have the same <code>name</code> attribute.",
         "Here's an example of a radio button:",
-        "<code>&#60;label&#62;&#60;input type=\"radio\" name=\"indoor-outdoor\"&#62; Indoor&#60;/label&#62;</code>",
+        "<code>&#60;label&#62;&#60;input type=\"radio\" name=\"indoor-outdoor\" for=\"indoor\" &#62; Indoor&#60;/label&#62;</code>",
```

Comment:
Some more changes:

1. The code sample needs the `for` attribute inside the `<label>`, NOT the `<input>`
2. Lets, split it into a block to make it cleaner. (refer: [Challenge Style Guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/seed/challenge-style-guide.md))
3. The clarification text is to be placed after the code sample.

```diff
-        "<code>&#60;label&#62;&#60;input type=\"radio\" name=\"indoor-outdoor\" for=\"indoor\" &#62; Indoor&#60;/label&#62;</code>",
+        "<blockquote>&#60;label for=\"indoor\"&#62; <br> &#60;input type=\"radio\" name=\"indoor-outdoor\"&#62;Indoor <br>&#60;/label&#62;</blockquote>",
+        "It is considered best practice to set the <code>for</code> attribute for assistive technologies to be able to link the relationship between the label and the child elements such as an <code>input</code> in this case.",
```